onnxruntime/core/platform/windows/env.cc

```diff
@@ -563,9 +705,42 @@ class WindowsEnv : public Env {
     return std::string();
   }
 
+  static std::vector<PROCESSOR_GROUP_INFO> GetProcessorGroups() {
+    DWORD returnLength = 0;
+    GetLogicalProcessorInformationEx(RelationGroup, nullptr, &returnLength);
+    auto last_error = GetLastError();
+    if (last_error != ERROR_INSUFFICIENT_BUFFER) {
+      return {};
+    }
+
+    std::unique_ptr<char[]> allocation = std::make_unique<char[]>(returnLength);
+    SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX* processorInfos = reinterpret_cast<SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX*>(allocation.get());
+
+    if (!GetLogicalProcessorInformationEx(RelationGroup, processorInfos, &returnLength)) {
+      last_error = GetLastError();
```

Comment:
> last_error = GetLastError();

Does this matter? You're returning in the very next line anyway.

Reply:
No, let's remove it.